### PR TITLE
DIKO pool cooldown

### PR DESCRIPTION
**Pull request by nieldeckx**
Contract code is ready to review. Will try to change the frontend this weekend.

### CODE REVIEW

**Review by philipdesmedt — APPROVED** [review]
Did a code review, no comments. Smart contracts look good.

Curious to see how you will implement this in the front-end :D

**Comment by nieldeckx** [comment]
Added the feature to the FE.

There is one TODO left. We need the current block-height to estimate how long it will still take before cooldown or redeem period ends. Do you know if we can fetch the current block-height in the FE using the Stacks library? 

Screenshot 2021-07-31 at 17 56 24

**Comment by philipdesmedt** [comment]
@nieldeckx yes - check the auctions.tsx file - I fetch the Stacks block height there.

**Comment by philipdesmedt** [comment]
```
      const client = getRPCClient();
      const response = await fetch(`${client.url}/v2/info`, { credentials: 'omit' });
      const data = await response.json();
      setStacksTipHeight(data['stacks_tip_height']);
```

**Comment by nieldeckx** [comment]
Great thx! Ready to review :)

File: clarity/tests/arkadiko-stake-pool-diko-v1-1_tests.ts [hunk 1]
```diff
@@ -124,11 +143,11 @@ async fn(chain: Chain, accounts: Map<string, Account>) {
       types.uint(100000000)
     ], wallet_1.address)
   ]);
-  block.receipts[0].result.expectOk().expectUint(475839436);
+  block.receipts[0].result.expectOk().expectUint(91428982948);
```
Comment (philipdesmedt) [inline clarity/tests/arkadiko-stake-pool-diko-v1-1_tests.ts:146]: @nieldeckx Q: why did this change? You know?

File: clarity/tests/arkadiko-stake-pool-diko-v1-1_tests.ts [hunk 2]
```diff
@@ -124,11 +143,11 @@ async fn(chain: Chain, accounts: Map<string, Account>) {
...
 
   // Check DIKO after unstaking
   call = chain.callReadOnlyFn("arkadiko-token", "get-balance", [types.principal(wallet_1.address)], wallet_1.address);
-  call.result.expectOk().expectUint(150375839436);  
+  call.result.expectOk().expectUint(241328982948);
```
Comment (philipdesmedt) [inline clarity/tests/arkadiko-stake-pool-diko-v1-1_tests.ts:150]: @nieldeckx Q: why did this change? You know?

File: clarity/tests/arkadiko-stake-pool-diko-v1-1_tests.ts (hunk 1 above)
Comment (nieldeckx) [reply clarity/tests/arkadiko-stake-pool-diko-v1-1_tests.ts:146]: @philipdesmedt Now you need to wait 10 days before you can unstake, so more rewards. See L135 where we advance 1450 blocks.

File: clarity/tests/arkadiko-stake-pool-diko-v1-1_tests.ts (hunk 2 above)
Comment (nieldeckx) [reply clarity/tests/arkadiko-stake-pool-diko-v1-1_tests.ts:150]: Same as other comment - we need to wait 10 days so we have more rewards when unstaking

**Pull request merged**